erikiado/jp2_online
# Task/delete photos

#### What's this PR do?
This PR, add the functionality to erase photos from the study. Also, it erases in the filesystem

#### Where should the reviewer start?
Check the receiver that its trigger on delete on estudios_socioeconomicos/models.py:111
Next, check the new button on jp2_online/templates/captura/panel_fotos/html
Then, validate the logic used check captura/views.py.
Finally, estudios_socioeconomicos/forms.py

#### How should this be manually tested?
You should make a new socioeconomical study, and go to the "Vivienda" section. On this site, upload some pictures. A delete button will appear below each photo; Click on any, and a modal will be shown, asking if you are sure of delete that picture. After you press yes, the picturo no longer will appear on the list. Validate that the picture was erase from the media folder.

#### Any background context you want to provide?
This is as per request by stakeholders.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
